File: daily-coding/coding-bat/python/not_string.py
# Given a string, return a new string where "not " has been added to the front. However, if the string already begins with "not", return the string unchanged. 

def not_string(str):
  front = str.split(' ')
  phrase = ''
  if front[0] == "not":
    phrase = str
  else:
    phrase = 'not ' + str
  return phrase

File: daily-coding/coding-bat/java/notString.java
// Given a string, return a new string where "not " has been added to the front. However, if the string already begins with "not", return the string unchanged. Note: use .equals() to compare 2 strings.

public String notString(String str) {

  String[] myArray = str.split(" ");

  if(myArray[0].equals("not")) {
    return str;
  }
  else {
    return "not " + str;
  }
}
